addon/utils/trigger.js

```diff
@@ -43,15 +54,17 @@ export default function trigger (elementOrSelector, eventName, isDraggingUp) {
 
 
 export async function sort (sourceList, sourceIndex, targetIndex, above, handleSelector) {
-  let sourceItem = sourceList.children[sourceIndex]
+  sourceList          = _getElementNode(sourceList)
+  let sourceListItems = _getListItems(sourceList)
+  let sourceItem      = sourceListItems[sourceIndex]
```

Comment:
As per the linter rule in this addon, it will throw error if semicolon is used. Since we are modifying 3rd party library, we will have to adhere by their linter rules & syntax. Thereby, it will not contradict if we raise a PR to the original library in future.